### DISCUSSION TITLE
Importing beatmaps from stable gets stuck

### DISCUSSION THREAD

**Discussion by BlauFx** [ppy/osu#19223] [Q&A] [answered]
### Type

Other

### Bug description

When importing maps from stable it gets stuck in this example at 1402. Checking logs indicates this might be related with aspire maps.

### Screenshots or videos

https://user-images.githubusercontent.com/34138240/179607791-86914315-598b-444f-ad44-cb843b14bd68.mp4

### Version

2022.719.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9135451/database.log)
[network.log](https://github.com/ppy/osu/files/9135452/network.log)
[performance.log](https://github.com/ppy/osu/files/9135453/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9135454/runtime.log)

**Comment by frenzibyte** [comment]
Please isolate this to the specific beatmap which is getting stuck, there's not enough information to work with here. Logs are just showing a bunch of random aspire maps failing and I'm not willing to spend time testing every one of them to figure that out (that's if I was able to get the exact copy you have).

**Reply by BlauFx** [reply]
Trying again to reimport beatmaps from stable somehow does not get stuck anymore.

**Comment by peppy** [comment, marked as answer]
Your imported folder was in a... fucked state.

The logs show one folder (`623665 Hommarju - HYENA`) containing thousands upon thousands of beatmaps.
